src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -369,6 +377,26 @@ export var analyticsActionCauseList = {
     type: 'facet',
     metaMap: { facetId: 1, facetValue: 2, facetTitle: 3 }
   },
+  categoryFacetSelect: <IAnalyticsActionCause>{
```

Comment:
It's possible to create a function that accepts a `name` param and returns an `IAnalyticsActionCause` object. It would be more concise since `type` and `metaMap` do not vary.